### PR TITLE
Refactor V619_2 edits to use dynamic dates

### PR DESCRIPTION
**Pull request by jaredb96**
Refactor V619_2 edits to use passed in dates. Also replaced and deleted old V619_2 edits and updated unit tests.

### CODE REVIEW

File: hmda/src/main/scala/hmda/validation/rules/lar/validity/2018/V619_2.scala
Comment (jaredb96) [inline hmda/src/main/scala/hmda/validation/rules/lar/validity/2018/V619_2.scala:1]: Note that these files are no longer needed

File: hmda/src/test/scala/hmda/validation/rules/lar/validity/2023/V619_2Spec.scala
Comment (jaredb96) [inline hmda/src/test/scala/hmda/validation/rules/lar/validity/2023/V619_2Spec.scala:1]: Never unit tested 2023 so I added it here

File: hmda/src/test/scala/hmda/validation/rules/lar/validity/2018/V619_2Spec.scala [hunk 1]
```diff
@@ -4,11 +4,10 @@ import hmda.model.filing.lar.LarGenerators._
 import hmda.model.filing.lar.LoanApplicationRegister
 import hmda.validation.rules.EditCheck
 import hmda.validation.rules.lar.LarEditCheckSpec
-import hmda.validation.rules.lar.validity._2018.V619_2
-
+import hmda.validation.rules.lar.validity.common.V619_2
 
 class V619_2Spec extends LarEditCheckSpec {
```
Comment (rkovalik-raft) [inline hmda/src/test/scala/hmda/validation/rules/lar/validity/2018/V619_2Spec.scala:9]: Can these tests be combined into a single test file in `...rules/lar/validity/common/V619_2Spec.scala` to check that all the filing years work as expected for V619_2? That way we won't have as much repetitive code to test each year

Comment (jaredb96) [reply hmda/src/test/scala/hmda/validation/rules/lar/validity/2018/V619_2Spec.scala:9]: Sure thing good catch

**Review by rkovalik-raft — APPROVED** [review]

**Pull request merged**